### PR TITLE
Define SUBUID_FILE/SUBGID_FILE

### PR DESCRIPTION
**Pull request by joakim-tjernlund**
These where hard coded, make them definable like SHADOW_FILE

### CODE REVIEW

**Comment by joakim-tjernlund** [comment]
Ideally there should be a configure option to define a common new prefix such as /etc/pwdb

File: lib/prefix_flag.c [hunk 1]
```diff
@@ -125,10 +125,10 @@ extern const char* process_prefix_flag (const char* short_opt, int argc, char **
 		spw_setdbname(spw_db_file);
 
 #ifdef ENABLE_SUBIDS
-		xasprintf(&suid_db_file, "%s/%s", prefix, "/etc/subuid");
+		xasprintf(&suid_db_file, "%s/%s", prefix, SUBUID_FILE");
```
Comment (ikerexxe) [inline lib/prefix_flag.c:128]: You forgot to remove `"` after `SUBUID_FILE`

Comment (joakim-tjernlund) [reply lib/prefix_flag.c:128]: ehh, so it seems. Wonder why I didn't see any mail with your comment

Comment (joakim-tjernlund) [reply lib/prefix_flag.c]: fixed now

**Review by ikerexxe — COMMENTED** [review]
LGTM! Merging

**Review by ikerexxe — APPROVED** [review]

**Pull request merged**